Library/Homebrew/test/rubocops/text_spec.rb

```diff
@@ -296,7 +292,7 @@ class Foo < Formula
       RUBY
     end
 
-    it "when deprecated `env :std` is present in homebrew-core" do
+    it "reports an offense if `env :std` is present" do
```

Comment:
```suggestion
    it "reports an offense if `env :std` is present in homebrew/core" do
```

Personally, I like having the `in homebrew/core`. If a change was made that causes this to fail, it wouldn't be immediately apparent that the offense should only be reported in homebrew/core. I could see someone fixing the test by making the offense always show when that isn't really what we want.